src/components/Todo.js

```diff
@@ -0,0 +1,53 @@
+import { useState } from 'react';
+import { nanoid } from 'nanoid';
+import { HiPlusCircle } from 'react-icons/hi';
+import TodoList from './TodoList';
+
+const Todo = () => {
+  const [data, setDate] = useState([]);
+  const [input, setInput] = useState('');
+
+  const handleChange = event => {
+    setInput(event.target.value);
+  };
+
+  const handleAdd = () => {
+    const newData = {
+      id: nanoid(),
+      title: input,
+      isChecked: false,
+    };
+    if (input === '') return;
+    setDate(prevData => [...prevData, newData]);
+    setInput('');
+  };
+
+  const handleCompleted = id => {
+    const completed = data.map(data => {
+      return data.id === id ? { ...data, isChecked: !data.isChecked } : data;
+    });
```

Comment:
The data argument name might crash with the data state declared at the top of the component. Consider changing to another name.